neqo-transport/src/packet/mod.rs

```diff
@@ -418,9 +418,22 @@ impl PacketBuilder {
             hex(hdr),
             hex(body)
         );
-        let ciphertext = crypto.encrypt(self.pn, hdr, body)?;
+
+        // Add space for crypto expansion.
+        let data_end = self.encoder.len();
+        for _i in 0..crypto.expansion() {
+            self.encode_byte(123);
+        }
```

Comment:
We can do better than this.  There is a `pad_to()` function that can be used.  Or you can define `pad_by()`.